core/controllers/questions_list.py

```diff
@@ -68,94 +68,72 @@ class QuestionsListHandler(
 
     @acl_decorators.open_access
     def get(self, comma_separated_skill_ids: str) -> None:
-        """Handles GET requests."""
-        assert self.normalized_request is not None
-        offset = self.normalized_request['offset']
-
-        skill_ids = list(set(comma_separated_skill_ids.split(',')))
-
-        try:
-            skill_fetchers.get_multi_skills(skill_ids)
-        except Exception as e:
-            raise self.PageNotFoundException(e)
-
-        question_summaries, merged_question_skill_links = (
-            question_services.get_displayable_question_skill_link_details(
-                constants.NUM_QUESTIONS_PER_PAGE + 1, skill_ids, offset)
-        )
-
-        if len(question_summaries) <= constants.NUM_QUESTIONS_PER_PAGE:
```

Comment:
Hey @shri3016, this PR contains superfluos changes. Please check!

Reply:
It looks like the new docstrings don't obey the lint rules for docstrings.

I'm going to close this PR, please run the lint tests locally and open a new PR when they pass on your machine. Make sure to follow the existing patterns when writing new docstrings. Thanks!